### ISSUE TITLE
Remove Recent Activity Query

### ISSUE THREAD

**Issue by k88hudson** [mozilla/activity-stream#1518]
Let's remove the Recent Activity query since it is no longer needed.

Note that this is blocked on the Highlights experiment being completed.

**Comment by Mardak** [comment]
Fixing with #1611.

**Comment by Mardak** [comment]
There's recent/frecent links and recent bookmarks that fed into top activity and activity feed.
